assets/elasticsearch/search.py

```
@@ -0,0 +1,47 @@
+
+collection_name = "elasticsearch_clp_bench"
+
+es = Elasticsearch(
+    "http://localhost:9202", request_timeout=30, max_retries=10, retry_on_timeout=True
+)
+results = []
+
+query = json.loads(sys.argv[1])
+
+# Function to execute a query without cache
+def execute_query_without_cache(query):
```

Ruff (0.12.2)

Missing return type annotation for public function `execute_query_without_cache`

Add return type annotation: `None`

(ANN201)